wasm_runtime/src/c_api.rs

```diff
@@ -468,3 +468,135 @@ pub extern "C" fn wasm_executionctx_run(executionctx_id: *const c_char, _buffer:
 pub extern "C" fn wasm_return_const_char_ownership(ptr: *const c_char) {
     deallocate_cstring(ptr);
 }
+
+
+// ##########################################################################
+//                              Unit Tests
+// ##########################################################################
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use std::ffi::CString;
+    use crate::config::WASM_RUNTIME_CONFIGS;
+
+    #[test]
+    fn wasm_wasm_module_load() {
+        // setup
+        let wasm_path = CString::new("../examples/wasm_modules/rust-wasm/hello_wasm.wasm").unwrap();
+        let nonexistent_path = CString::new("../attempt/to/load/me.wasm").unwrap();
+
+        // test
+        let mut val = wasm_module_load(wasm_path.as_ptr());
+        assert_eq!(val, 0);
+
+        val = wasm_module_load(nonexistent_path.as_ptr());
+        assert_eq!(val, -1);
+    }
+
+    #[test]
+    fn wasm_wasm_config_create() {
+        // setup
+        let config_str = "Wordpress";
```

Comment:
I'd use a more neutral value for `config_str`, such as the one used below (eg: "Hello Wasm").

Reply:
This has been corrected for all tests referencing "Wordpress"